docs/topics/modules/creating_new_application.adoc

```diff
@@ -20,83 +20,68 @@ ifeval::["{context}" == "hello-world"]
-+
-image::authorize_github_screen.png[Authorize GitHub]
-+
-. The next screen displays a summary of your application options. Scroll down in your browser to view the *Application Information* section. For this example, do not edit these options. If desired, you can change the project name, version, Group ID, which space it is in, and the target environment for your new application at this step.
-+
-image::{context}_app_information.png[Application information]
-+
-. Click btn:[Set Up Application] to finalize your choices and create the new application.
-
-. The progress screen displays a confirmation message when your application is ready.
-.. Optionally, click the blue link to view your new codebase in your Git provider.
-.. When ready, click btn:[View New Application].
-+
-image::{context}_application_ready.png[Application ready]
-+
-Your new Vert.X project is now created in your space and your space dashboard now displays your new application:
-+
-image::space_dash_after_hello-world.png[Space dashboard view after creating Hello World]
-+
-. You have now completed the task, *Launch a sample quickstart application*, in the *Test Iteration*. Ensure that you change the state of the work item to *Closed* using the *Plan* tab
-
 endif::[]
 
-
 // for user-guide
 ifeval::["{context}" == "user-guide"]
 . In the *Name your application* field, type a unique name for your new project. Ensure that the application name adheres to the listed *Naming Requirements*.
 
-+
-image::app_name_requirements.png[Naming Requirements]
-+
-
 . Select the *Create a new codebase* radio button and click btn:[Continue].
 
 . Select the mission and runtime for your new project:
 
 .. In the *Choose a mission* section, select the the appropriate option.
-
-.. In the *Choose a runtime* section, select the appropriate runtime. When you select the options at each step, the gray arrow at the bottom of the screen turns blue.
-
+.. In the *Choose a runtime* section, select the appropriate runtime. The gray arrow at the bottom of the screen now turns blue.
 .. Click the blue downward arrow button to continue.
 +
-image::{context}_choose_mission_runtime.png[Choose mission and runtime]
-+
-. In the *Select Pipeline* section, select the appropriate option, then click the blue arrow to continue to the next step. We recommend using the first option for most use cases because it provides stages to test your changes for each pipeline build. For more information see <<working_with_pipelines>>.
+image::choose_mission_runtime.png[Choose mission and runtime]
 +
-image::user-guide_select_pipeline.png[Select a pipeline]
+. In the *Select Pipeline* section, select the appropriate option, then click the blue arrow to continue to the next step. We recommend selecting the first option for most use cases because it provides stages to test your changes for each pipeline build. For more information see <<working_with_pipelines>>.
```

Comment:
We're using recommend here again but changes to suggest above. I'd "recommend" picking one or the other as the thing we tell users to do and use that consistently across the docs.